### ISSUE TITLE
Error when launching

### ISSUE THREAD

**Issue by DvagNic** [astefanutti/kubebox#55]
When I try to run kubebox (on the Mac), I get the following error:
```
ReferenceError: os is not defined
    at new OpenIdConnect (/kubebox/client.js:186:7)
    at Client.set master_api [as master_api] (/kubebox/client.js:438:30)
    at new Kubebox (/kubebox/client.js:1705:27)
    at Object.1../lib/config/context (/kubebox/client.js:42:1)
    at o (/kubebox/client.js:1:327)
    at r (/kubebox/client.js:1:493)
    at Object.<anonymous> (/kubebox/client.js:1:522)
    at Module._compile (module.js:660:30)
    at Object.Module._extensions..js (module.js:671:10)
    at Module.load (module.js:573:32)
```

The same happens when I use the docker image.

**Comment by astefanutti** [comment]
Thanks a lot for the report. We'll fix it in the upcoming release.

**Comment by astefanutti** [comment]
It should be fixed with d68ffb0a660a3026f8ccbcfb569ca75c81f198ec.

We plan to cut a release ASAP. In the meantime, you should be able to test the fix by running the `latest` tag of the Docker image.

Thanks again for the report.
